paddle/fluid/framework/ir/mkldnn/conv_activation_mkldnn_fuse_pass.cc

```diff
@@ -14,110 +14,104 @@
 
 #include "paddle/fluid/framework/ir/mkldnn/conv_activation_mkldnn_fuse_pass.h"
 
-#include <vector>
-
 #include "paddle/fluid/framework/op_version_registry.h"
-#include "paddle/fluid/platform/enforce.h"
-
-namespace paddle {
-namespace framework {
-class OpDesc;
-}  // namespace framework
-}  // namespace paddle
+#include "paddle/fluid/string/pretty_log.h"
 
 namespace paddle {
 namespace framework {
 namespace ir {
 
-class Graph;
+using string::PrettyLogDetail;
+
+void ConvActivationMkldnnFusePass::ApplyImpl(Graph* graph) const {
+  std::vector<std::string> act_types = {
+      "relu", "mish",  "swish",   "hard_swish", "hard_sigmoid",
+      "gelu", "relu6", "sigmoid", "leaky_relu"};
+
+  std::vector<std::string> conv_types = {"conv2d"};
 
-void ConvActivationFusePass::ApplyImpl(ir::Graph* graph) const {
+  for (const auto& conv_type : conv_types)
+    for (auto& act_type : act_types) {
+      std::unordered_map<std::string, std::string> attrs_map;
+
+      if (act_type == "swish")
+        attrs_map.emplace("beta", "fuse_alpha");
+      else if (act_type == "relu6")
+        attrs_map.emplace("threshold", "fuse_alpha");
+      else if (act_type == "hard_sigmoid") {
+        attrs_map.emplace("slope", "fuse_alpha");
+        attrs_map.emplace("offset", "fuse_beta");
+      } else {
+        attrs_map.emplace("alpha", "fuse_alpha");
+        attrs_map.emplace("beta", "fuse_beta");
+      }
+      FuseConvAct(graph, conv_type, act_type, attrs_map);
+    }
+}
+
+void ConvActivationMkldnnFusePass::FuseConvAct(
+    Graph* graph, const std::string& conv_type, std::string& act_type,
+    const std::unordered_map<std::string, std::string>& attrs_map) const {
   PADDLE_ENFORCE_NOT_NULL(
       graph, platform::errors::InvalidArgument("Graph cannot be nullptr."));
-  FusePassBase::Init("conv_activation_mkldnn_fuse", graph);
+  FusePassBase::Init(conv_type + "_" + act_type + "_mkldnn_fuse_pass", graph);
```

Comment:
Hi, ok so here the passes with different activations will have different names and AddStatis belongs to different passes. That's why CHECK_EQ(fuse_status.at("conv2d_gelu_mkldnn_fuse_pass"), 2); can find correctly.
But it is fuse_statis not fuse_status. But that is a small thing.

Reply:
I didn't notice this typo, thank you!